spec/integration/helpers/start-atom.js

```diff
@@ -59,9 +59,6 @@ const buildAtomClient = async (args, env) => {
   // So, for now, we'll just use a random port (we don't care about its value since we're not
   // connecting through it).
   // (inspired by https://github.com/electron/spectron/pull/361/commits/737db138bd8a6daaf80f9c2bff710ce4a5fff39b).
-  // TodoElectronIssue: Remove the whole remote-debugging-port param once we upgrade
-  // to Electron v5, since this was fixes there (see electron/electron#17800).
-  const randomPort = Math.floor(Math.random() * (9999 - 9000) + 9000);
```

Comment:
I think this entire comment is describing the need for the workaround. I think the whole comment can be deleted given that the workaround is apparently not needed anymore.

Reply:
(I wasn't able to highlight the whole comment, but I meant all eight lines of the comment.)

Reply:
I believe the first part of the comment is still relevant. Because there is still a random port set-up for the debugger. The only thing is that the problem of with it being set to zero (which is the default value), which means to make it random, was fixed in the newer Electron releases.

Reply:
If you visit the PR mentioned, the discussion there is strictly based on solving the ChromeDriver issue this whole comment is talking about.

https://github.com/electron/electron/pull/17800

> Notes: Fixed issue where `chromedriver` would not connect correctly if you didn't provide a custom `remote-debugging-port`

Electron sets this random port for us, to satisfy ChromeDriver's rather arbitrary requirements. _(Knowing about this is not important to Atom developers, as long as ChromeDriver generally behaves itself. According to the comment, we don't actually use this port for anything, other than (one infers) letting ChromeDriver manage and use this port in a fully automated way.)_ So the behavior described in the comment lives in Electron now, not Atom's codebase. With the fix landed at Electron, all of this is just acting with the expected/default behaviors, which doesn't require a comment to describe it at this repo.

It's interesting to have an insight into what the code is doing, or historical context for why the code is the way it is, and more comments are usually better, but I think the convention is to not have comments for things that are working the usual/expected way, and for things that don't require special attention from devs.

In other words, this isn't actionable information anymore, so encountering this comment might mislead readers that there is something they have to do about this issue, or some care they need to take. In fact, it was effectively a bug in upstream Electron, which at one point Atom had to work around, but which has since been fixed. If the workaround is deleted, the comment justifying it logically should be cleaned up along with the workaround.

To the extent that it remains relevant, the comment "lives on" in the commit history of this file. _(If delving into the past, any place folks encounter this workaround back in the commit history, the comment will be preserved there along with it.)_

Reply:
You are right. Since I am just cleaning it up, I thought that the first part might still be relevant for Atom devs too, as I thought using ChromeDriver was non-trivial. But if it is, than it is indeed not necessary information. I will update the PR